coverage/lcov-report/block-navigation.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+var jumpToCode = (function init() {
+    // Classes of code we would like to highlight in the file view
```

Comment:
- [ ] [OPTIONAL]  You write too many comments in your codebase. Note that comments that add no information have a negative value. Please remove comments you don't really need in your code. I suggest you follow [Comment Best Practices](https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/) :+1: